front/src/dashboard/components/BSDList/BSDD/WorkflowAction/WorkflowAction.tsx

```diff
@@ -48,11 +50,13 @@ export function WorkflowAction(props: WorkflowActionProps) {
       return null;
     }
     case FormStatus.Sent: {
-      if (siret === form.recipient?.company?.siret) {
-        if (isTempStorage) {
-          return <MarkAsTempStored {...props} />;
+      if (isActTab) {
```

Comment:
Est-ce que tu pourrais faire l'update coté V2 aussi stp ? Apps/Dashboard/dashboardServices.ts ligne 323
`if (isActTab && isBsdd(bsd.type)) {
    if (isSameSiretDestination(currentSiret, bsd) && bsd.isTempStorage) {
      return VALIDER_ENTREPOSAGE_PROVISOIRE;
    }
    return VALIDER_RECEPTION;
  }`

Ou sinon je le ferais dans une autre PR as you want